lib/inspec/fetcher/local.rb

```diff
@@ -3,7 +3,7 @@
 module Inspec::Fetcher
   class Local < Inspec.fetcher(1)
     name "local"
-    priority 0
+    priority 1
```

Comment:
@Nik08 just in case if you can add some dev docs / code comments here explaining about what is this priority setting does that will be great.

Reply:
@Vasu1105 I will add the comments in this file since priority is used in a lot of libraries and it means different for each case. So I will mention how this was important in comments.